editor/editor_node.cpp

```diff
@@ -6007,6 +6024,15 @@ EditorNode::EditorNode() {
 	p->add_shortcut(ED_SHORTCUT("editor/project_settings", TTR("Project Settings...")), RUN_SETTINGS);
 	p->connect("id_pressed", this, "_menu_option");
 
+	vcs_actions_menu = VersionControlEditorPlugin::get_singleton()->get_version_control_actions_panel();
+	vcs_actions_menu->set_name("Version Control");
+	vcs_actions_menu->connect("index_pressed", this, "_version_control_menu_option");
+	p->add_separator();
+	p->add_child(vcs_actions_menu);
+	p->add_submenu_item(TTR("Version Control"), "Version Control");
+	vcs_actions_menu->add_item(TTR("Set Up Version Control"), RUN_VCS_SETTINGS);
```

Comment:
As "Set Up Version Control" opens a dialog rather than performing an immediate action, my understanding is that the menu option would be "Set Up Version Control...".

Reply:
I didn't know this was one of the UI guidelines while making this. I think I understand the difference between the two. Maybe this could be a separate issue